beginner_source/basics/intro.py

```diff
@@ -34,7 +34,7 @@
 ------------------
 You can run this tutorial in a couple of ways:
 
-- **In the cloud**: This is the easiest way to get started! Each section has a "Run in Microsoft Learn" link at the top, which opens an integrated notebook in Microsoft Learn with the code in a fully-hosted environment.
+- **In the cloud**: This is the easiest way to get started! Each section has a "Run in Microsoft Learn" and "Run in Google Colab" link at the top, which opens an integrated notebook in Microsoft Learn (Microsoft account required) or Google Colab (no account required), respectively, with the code in a fully-hosted environment.
```

Comment:
LGTM, but can we remove the part about accounts please? This is something Microsoft and Google control and can change any time and we might miss.

Reply:
Done. Needing a Microsoft account (and logging into it) was a big hassle for me, so I was trying to give learners a hint about that, but I appreciate your point.